### DISCUSSION TITLE
Please ignore accents (áéíóú) in variable names.

### DISCUSSION THREAD

**Discussion by lsholguin** [kazad/instacalc#102] [Ideas]
Variable names are case independent.  Please "normalize" further by ignoring accents.
When using IC in Spanish, it will be kindly apreciated.

**Comment by kazad** [comment]
Great suggestion! Updated: https://instacalc.com/57660

```
Café = 10      = 10
cafe * 10      = 100
```

**Reply by lsholguin** [reply]
Great!

Thank you very much, Kalid.
